doc/man/scons.xml

```diff
@@ -1125,11 +1125,12 @@ the mechanisms in the specified order.</para>
         the special tokens <literal>all</literal> or <literal>none</literal>.
         A comma-separated string can be used to select multiple features.
         The default setting is <literal>none</literal>.</para>
-      <para>Current available features are: <literal>ninja</literal>.</para>
+      <para>Current available features are: <literal>ninja</literal>, <literal>tm_v2</literal>.</para>
       <caution><para>
         No Support offered for any features or tools enabled by this flag.
       </para></caution>
-      <para><emphasis>Available since &scons; 4.2.</emphasis></para>
+        <para><emphasis><literal>ninja</literal> Available since &scons; 4.2.</emphasis></para>
+        <para><emphasis><literal>tm_v2</literal> Available since &scons; 4.4.1</emphasis></para>
```

Comment:
Aside: we should define a placeholder value that uses a pattern the release tools recognize. When it's time to make a release, it can flag there are files that need the actual version updated if that pattern is found. For now we'll just have to remember that the placeholder is "4.4.1".

Reply:
yes. perhaps __NEXT_VERSION__ ?